### DISCUSSION TITLE
Kafka/Redpanda for queuing otel data

### DISCUSSION THREAD

**Discussion by spsneo** [uptrace/uptrace#326] [General]
I want to use redpanda for queuing otel data before it gets stored in the clickhouse. Is there any documentation around it?

**Comment by vmihailenco** [comment]
Uptrace Community edition does not support Kafka. That feature is only supported in [Uptrace Enterprise](https://uptrace.dev/get/enterprise.html#optimized-storage)

**Reply by spsneo** [reply]
Got it. 
Is there any pointer if I want to do it myself?

**Reply by vmihailenco** [reply]
That's a pretty big feature. Besides, most people don't want to have an extra dependency so it would need to be optional which would make it even more complex.

I don't think this is something we want to support in the Community Edition.

**Reply by spsneo** [reply]
Thanks for your inputs.
